### PR TITLE
fix: AMP violation on Account summaries page(CXSPA-1884)

### PR DESCRIPTION
**Pull request by kimhw0630**
Closes: https://jira.tools.sap/browse/CXSPA-1884

Added missing aria-label for icon buttons

### CODE REVIEW

File: feature-libs/organization/administration/components/shared/list/list.component.html [hunk 1]
```diff
@@ -15,6 +15,7 @@ <h3>
                 appendToBody: true,
                 displayCloseButton: true
               }"
+              [attr.aria-label]="'organization.hint' | cxTranslate"
```
Comment (justinlee01) [inline feature-libs/organization/administration/components/shared/list/list.component.html]: Is "hint" really adequate for an aria-label?

**Review by justinlee01 — DISMISSED** [review]

**Review by bullfroggy-sap — DISMISSED** [review]

File: feature-libs/organization/administration/components/shared/list/list.component.html (hunk 1 above)
Comment (kimhw0630) [reply feature-libs/organization/administration/components/shared/list/list.component.html]: I was deciding between hint vs. information but popup content is coming from the hint field so I decided to use hint...

File: feature-libs/organization/administration/assets/translations/en/organization.ts [hunk 2]
```diff
@@ -113,6 +113,7 @@ export const organization = {
         unknown: 'Server validation error.',
       },
     },
+    hint: 'hint',
```
Comment (ercultimate) [inline feature-libs/organization/administration/assets/translations/en/organization.ts]: Is this a resonable default value?

File: feature-libs/organization/administration/components/shared/list/list.component.html (hunk 1 above)
Comment (kimhw0630) [reply feature-libs/organization/administration/components/shared/list/list.component.html]: UX told me to use "information" so I will update

File: feature-libs/organization/administration/assets/translations/en/organization.ts (hunk 2 above)
Comment (kimhw0630) [reply feature-libs/organization/administration/assets/translations/en/organization.ts]: change to `information`

**Review by Zeyber — APPROVED** [review]

**Pull request merged**